src/Components/Bookshelf/bookshelf.css

```diff
@@ -0,0 +1,15 @@
+.bookshelf {
+    max-width: 1300px;
+    margin: 0 auto;
+    padding: 3rem;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+    gap: 2rem;
+}
```

Comment:
This needs a small amount of work on responsivity

Reply:
done